bin/pycbc_live

```diff
@@ -378,13 +379,17 @@ with ctx:
                     logging.info("PSD outside acceptable sensitivity %s - %s, have %s",
                                  args.min_psd_abort_distance, args.max_psd_abort_distance, dist)
                     status = False               
-    
+
+            if ifo in followup_ifos:
```

Comment:
In response to the issue you shown me, perhaps add an explicit buffer length option? I'm not sure I want to set it based on the template bank as that forces everyone to generate the full bank just to check (i.e. not easy). It's not usually complicated to guess, so why not allow an override option for this case.